instructor_notes/openscpca/openscpca-setup.md

```diff
@@ -0,0 +1,86 @@
+
+The goal for this activity is to set everyone up on LSfR, but we if internet connectivity is too poor, we can focus on the local setup steps:
+
+1. We will begin by presenting a set of slides outlining all setup steps
+   - Slides will also have explained differences between local setup and LSfR setup
+
+2. Direct participants to [fork](https://openscpca.readthedocs.io/en/latest/technical-setup/fork-the-repo/) `OpenScPCA-analysis`
+
+3. Sign into LSfR and create an instance:
+    - Sign in using the link you were emailed
+    - Navigate to Service Catalog
+    - [Create a Standard-2XL VSCodium LSfR instance](https://openscpca.readthedocs.io/en/latest/software-platforms/aws/creating-vcs/)
+      - Explain that we are using VSCodium since we plan to write a Python module, but we could launch an RStudio instance for R development as well.
+      Similarly, if we anticipated needing more computational resources, we could select different instance options.
+
+4. While the instance is being created (~10 minutes), lead participants through some _local setup_:
+
+  - [Set up conda](https://openscpca.readthedocs.io/en/latest/technical-setup/environment-setup/setup-conda/) (assuming it has already been installed):
+    ```bash
+    # set up channels
+    conda config --add channels defaults
+    conda config --add channels bioconda
+    conda config --add channels conda-forge
+    conda config --set channel_priority strict
+
+    # install packages into base
+    conda install awscli conda-lock jq pre-commit
+    ```
+
+  - [Configure AWS CLI](https://openscpca.readthedocs.io/en/latest/technical-setup/environment-setup/configure-aws-cli/) (may wish to show this website during configuration for prompt responses)
+    ```bash
+    # launch configuration and fill in prompts
+    aws configure sso
+
+    # after configuration, log in (this will launch a browser)
+    aws sso login --profile openscpca
+    ```
+
+  - If we have time and/or it's clear we need to work locally, they can also clone their `OpenScPCA-analysis` fork, and [set up pre-commit](https://openscpca.readthedocs.io/en/latest/technical-setup/environment-setup/setup-precommit/) via the GitKraken terminal
+
+5. Back on LSfR, [create a volume](https://openscpca.readthedocs.io/en/latest/software-platforms/aws/working-with-volumes/) and attach it to the VSCodium instance.
+
+6. Finally, we can [launch the VSCodium instance](https://lfr.console.aws.amazon.com/ls/research/webapp/home/virtual-computers) and set up:
+    - First, launch from the "Access operating system" dropdown
+      - Set up the [conda environment](https://openscpca.readthedocs.io/en/latest/software-platforms/aws/starting-development-on-lsfr/#create-and-activate-a-conda-environment):
+      !! REVIEWER TODO: We are recommending installing into an openscpca environment here but into base environment for local setup. Do we want to change any docs?
```

Comment:
I think we probably do want to go back and reconsider the docs... I think we might have the docs follow the model here, but also show an option to install the packages into the `base` environment with (I think) `conda env update  -n base -f environment.yml`

Reply:
I opened an issue for this in `OpenScPCA-analysis` as part of the OpenRRP epic.